assets/src/edit-story/components/layout/index.js

```diff
@@ -60,6 +60,20 @@ const Editor = withOverlay(styled.section.attrs({
     minmax(${LIBRARY_MIN_WIDTH}px, ${LIBRARY_MAX_WIDTH}px)
     minmax(${CANVAS_MIN_WIDTH}px, 1fr)
     minmax(${INSPECTOR_MIN_WIDTH}px, ${INSPECTOR_MAX_WIDTH}px);
+
+  ::before {
+    content: '';
+    position: absolute;
+    border-style: solid;
+    border-width: 1px 0 0 1px;
+    border-color: ${({ theme }) => theme.colors.divider.secondary};
+    left: 0;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    pointer-events: none;
+    z-index: ${({ zIndex }) => zIndex};
+  }
```

Comment:
That's a lot of CSS for a simple border.

Can't we just add

```css
border-left: 1px solid rgba(255, 255, 255, .8); /* theme.colors.divider.secondary */
border-top: 1px solid rgba(255, 255, 255, .8); /* theme.colors.divider.secondary */
```

here:

https://github.com/google/web-stories-wp/blob/8c3a778e0cedc2360b567caa3c1b5338ba13c7e5/assets/src/edit-story/style.css#L38-L42

?